pkg/kubectl/describe/describe.go

```diff
@@ -17,10 +17,43 @@ limitations under the License.
 package describe
```

Comment:
Rename this file to `interfaces.go` it'll work similar to what we have in polymorphichelpers.